.idems_app/deployments/plh/default.config.ts

```diff
@@ -19,5 +19,6 @@ const config: IDeploymentConfig = {
     translated_strings_path: "packages/app-data/translations/from_translators",
     source_strings_path: "packages/app-data/translations/to_translate",
   },
+  workflows: ["default.workflows.ts"],
```

Comment:
Workflows have been designed so that they can be imported across multiple files within any deployment